src/tools/illink/src/ILLink.Shared/DiagnosticId.cs

```diff
@@ -187,6 +187,7 @@ public enum DiagnosticId
 		_unused_DynamicallyAccessedMembersOnTypeReferencesCompilerGeneratedMemberOnBase = 2120,
 		RedundantSuppression = 2121,
 		TypeNameIsNotAssemblyQualified = 2122,
+		_EndTrimAnalysisWarningsSentinel = 2123,
```

Comment:
Would it be better to not specify the value - AFAIK C# should assign the next (+1) value automatically which would make this work nicely when we add a new one.